### PR TITLE
[ADD]  travis: add openerp_template logs to stdout.log

### PR DESCRIPTION
**Pull request by fernandahf**
Description of the issue/feature this PR addresses:

Logs generated in openerp_template process is not saved in stdout.log.

Current behavior before PR:
Logs generated in openerp_template process lost.

Desired behavior after PR is merged:
Logs generated in openerp_template saved in stdout.log

-- I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: travis/test_server.py [hunk 1]
```diff
@@ -260,16 +261,19 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
...
-            if line == b'':
-                break
-            if hidden_line(line, main_modules=[], addons_path_list=[],
-                           hidden_all_no_translation=True):
-                continue
-            # No show a warning from template runtime
-            # if 'openerp.models.schema' in line:
-            #     line = line.replace('DEBUG', 'WARNING')
-            print(print_log(line))
+        if stdout_log:
```
Comment (moylop260) [inline travis/test_server.py:264]: Could you test with another project without configure if the log is printed?

Comment (fernandahf) [reply travis/test_server.py:264]: I tested with a gitlab project, modified MQT branch and showed me logs openerp_template, also I put a pdb and checked that was in this line printing logs.

Comment (moylop260) [reply travis/test_server.py:264]: Could you share the URL of the pipeline, please?

NOTE: It should be different to Absa

Comment (fernandahf) [reply travis/test_server.py:264]: Of course,

https://git.vauxoo.com/vauxoo-dev/lodi/-/pipelines/111097

Regards.

**Pull request merged**